pkg/status/cel-evaluator.go

```diff
@@ -37,7 +46,9 @@ type celEvaluator struct {
 func newCELEvaluator() (*celEvaluator, error) {
 	env, err := cel.NewEnv(
 		cel.Declarations(
-			decls.NewVar("obj", decls.NewMapType(decls.String, decls.Dyn)),
+			decls.NewVar(sourceObjectKey, decls.NewMapType(decls.String, decls.Dyn)),
+			decls.NewVar(returnedKey, decls.NewMapType(decls.String, decls.Dyn)),
+			decls.NewVar(inventoryKey, decls.NewMapType(decls.String, decls.String)),
```

Comment:
I suspect it is possible to say here that this map has only one string in its domain, `"name"`. I think it would be good to say that, at least by the time this code is developed enough to shift the type checking left. Which is not yet.